pom.xml

```diff
@@ -144,6 +144,16 @@
       <artifactId>workflow-support</artifactId>
       <classifier>tests</classifier>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.jboss.marshalling</groupId>
+          <artifactId>jboss-marshalling-river</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.jboss.marshalling</groupId>
+          <artifactId>jboss-marshalling</artifactId>
```

Comment:
Hmm, why was this necessary?

Reply:
Not sure but it seems it was pulled in packaging according to build logd

Reply:
I tracked it down to the addition of workflow-cps in https://github.com/jenkinsci/kubernetes-plugin/pull/707 which is pulling workflow-support transitively and then altering the scope of following dependencies (from `test` to `compile`)

Reply:
An occurrence of https://github.com/jenkinsci/maven-hpi-plugin/pull/140#discussion_r322739625 essentially

Reply:
Trying out a new fix of maven-hpi-plugin

Reply:
https://github.com/jenkinsci/maven-hpi-plugin/pull/172/